nextjs-task-system/actions/notifications/notifications-actions.ts

```
@@ -0,0 +1,77 @@
+  }
+}
+
+export async function saveMessageAndTask(
+  formattedTask: Task,
+  token: string,
+  messageForWebsocket: string,
+) {
+  try {
+    {
+      /*Logic to save notifications by user*/
+    }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)